### PR TITLE
Bump jekyll version to 3.8.5

### LINKED ISSUE #555: update to Jekyll 3.8
Is it plan to update to Jelyll 3.8? Currently 3.7.3 is used. :no_mouth: An update would be great!

### PR DESCRIPTION
**Pull request by merlinpatt**
This is to fix the issue of being unable to use include_relative inside of collections ([jekyll issue 6816](https://github.com/jekyll/jekyll/issues/6816)). This will also fix #555

### CODE REVIEW

**Comment by merlinpatt** [comment]
@pathawks done. I didn't even realize it was out, my bad

**Comment by merlinpatt** [comment]
Any word on this?

File: lib/github-pages/dependencies.rb [hunk 1]
```diff
@@ -7,7 +7,7 @@ module GitHubPages
   class Dependencies
     VERSIONS = {
       # Jekyll
-      "jekyll"                    => "3.7.3",
+      "jekyll"                    => "3.8.2",
```
Comment (kenman345) [inline lib/github-pages/dependencies.rb]: Jekyll 3.8.3? https://github.com/jekyll/jekyll/releases/tag/v3.8.3

**Comment by kenman345** [comment]
V3.8 of Jekyll brings a few changes that really help speed up build times. It would be great if we could get the latest and greatest on this gem so github pages users can experience those performance increases.

**Comment by merlinpatt** [comment]
@parkr is there something holding this up? What can I do to help move this along?

**Comment by ashmaroli** [comment]
Its `v3.8.5` now..

**Comment by slgraff** [comment]
Is there any update to this? We've had a few users contact support asking about the 3.8.x version and when it will be available on Pages.

**Comment by DirtyF** [comment]
Duplicate of #596

**Comment by RDIL** [comment]
@parkr anything blocking this?

**Comment by RDIL** [comment]
@parkr can we get this update pushed out before jekyll 4.0 comes out?  Just think it will make the upgrade a little more smooth.

**Comment by RDIL** [comment]
Hooray!

**Pull request merged**